WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -243,41 +234,19 @@ public Wallet AddWallet(KeyManager keyManager)
 
 	private Wallet LoadWalletByNameFromDisk(string walletName)
 	{
-		(string walletFullPath, string walletBackupFullPath) = WalletDirectories.GetWalletFilePaths(walletName);
-		Wallet wallet;
+		string walletFullPath = WalletDirectories.GetWalletFilePaths(walletName);
 		try
 		{
-			wallet = _walletFactory.Create(KeyManager.FromFile(walletFullPath));
+			return _walletFactory.Create(KeyManager.FromFile(walletFullPath));
 		}
 		catch (Exception ex)
 		{
-			if (!File.Exists(walletBackupFullPath))
-			{
-				throw;
-			}
-
 			Logger.LogWarning($"Wallet got corrupted.\n" +
 				$"Wallet file path: {walletFullPath}\n" +
-				$"Trying to recover it from backup.\n" +
-				$"Backup path: {walletBackupFullPath}\n" +
 				$"Exception: {ex}");
-			if (File.Exists(walletFullPath))
-			{
-				string corruptedWalletBackupPath = $"{walletBackupFullPath}_CorruptedBackup";
-				if (File.Exists(corruptedWalletBackupPath))
-				{
-					File.Delete(corruptedWalletBackupPath);
-					Logger.LogInfo($"Deleted previous corrupted wallet file backup from `{corruptedWalletBackupPath}`.");
-				}
-				File.Move(walletFullPath, corruptedWalletBackupPath);
-				Logger.LogInfo($"Backed up corrupted wallet file to `{corruptedWalletBackupPath}`.");
-			}
-			File.Copy(walletBackupFullPath, walletFullPath);
 
-			wallet = _walletFactory.Create(KeyManager.FromFile(walletFullPath));
```

Comment:
I don't understand why we tried the same thing in the try and the catch block before, but now I just throw the exc out to the user.